src/docs/release-notes/R22/R22.mdx

```diff
@@ -15,8 +15,12 @@ The Power Xpert Blue "PX Blue" design system rebranded to Brightlayer User Inter
 ### Documentation Site
 
 -   Added [migration instructions](https://brightlayer-ui.github.io/migration) for package migration
--   Added new design patterns for Angular and React
--   Updated design patterns to be responsive
+
+### Design Pattern Examples
```

Comment:
I assume you have a specific reason for not listing "Design Pattern Examples" under "Documentation Site"... I don't think people would know what this section is referring to from just the title.

Reply:
Yeah, we did have it under the documentation section and we didn't want readers to assume that these were just documentation on doc'it. We can put any label on it. What do you suggest

Reply:
interactive design pattern examples (they are called "interactive examples" on docit), design pattern code examples, etc